src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder results = new StringBuilder();
+        String importantSymbol = ",";
+        int operationTypeIndex = 0;
+        int amountIndex = 1;
+        int arraysTheoryMaxSize = 10;
+        int realSize = 0;
```

Comment:
The variable `arraysTheoryMaxSize` is unnecessary. Instead of using a fixed size array, consider using a `Map` to dynamically store the operation types and their amounts.

Reply:
but from lerning program i didn't have anything about map. it suppose to be layter